utils/idmloader/dfns.txt

```diff
@@ -0,0 +1,59 @@
+# Add a new DFN parameter set to MF6
+# by adding a new entry to this list
+# ----------------------------------
+
+# simulation
+sim-nam.dfn
+
```

Comment:
sim-tdis.dfn should be here?  Also, it is an outlier with respect to the naming pattern- should probably be sim-tdisidm.f90 if possible.